src/views/Pools/components/CardFooter.tsx

```diff
@@ -133,6 +152,16 @@ const CardFooter: React.FC<Props> = ({
               <Balance fontSize="14px" isDisabled={isFinished} value={blocksRemaining} decimals={0} />
             </Row>
           )}
+          {isMetamask && (
+            <FlexFull>
```

Comment:
You probably can remove this wrapper, and add the property `flex={1}` on the child https://styled-system.com/api#flexbox

Reply:
fixed